eng/pipelines/extra-platforms/runtime-extra-platforms-wasm.yml

```diff
@@ -133,9 +117,6 @@ jobs:
       isWasmOnlyBuild: ${{ parameters.isWasmOnlyBuild }}
       # Always run for runtime-wasm because tests are not run in runtime
       alwaysRun: ${{ parameters.isWasmOnlyBuild }}
-
-      # NOTE - Since threading is experimental, we don't want to block mainline work
-      shouldContinueOnError: true
```

Comment:
We should keep this for a couple of weeks and watch if the job stays green. If it does, then we can drop this.